Formula/qodem.rb

```diff
@@ -15,8 +16,6 @@ class Qodem < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "19414f9b69d7423a5f2ab1ecb2690bf1bd285f96832d151835741867f22f2e6e"
   end
 
-  deprecate! date: "2022-05-07", because: :repo_removed
```

Comment:
The repo is archived, which we also consider deprecated (`:repo_archived`). Unless there is a maintained repo, this should remain deprecated.